R/BatchtoolsParam-class.R

```diff
@@ -369,6 +382,21 @@ setMethod("bplapply", c("ANY", "BatchtoolsParam"),
         file.copy(logs, bplogdir(BPPARAM) , recursive=TRUE, overwrite=TRUE)
     }
 
+    ## WARNING
+    ## Save a registry in a folder with extension, _saved_registry
+    ## BatchtoolsParam('saveregistry=TRUE') option should be set only
+    ## when debugging. This can be extremely
+    if (.bpsaveregistry(BPPARAM)) {
+        message(
+            "'BatchtooParam(saveregistry = TRUE)' is being used.\n",
```

Comment:
this info is better presented in the man page (it's already there, good!); remove from here